m4/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "m4"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { version = "4", features = ["cargo"] }
+nom = "7.1.3"
+libc = "0.2.153"
+once_cell = "1.19.0"
+thiserror = "1.0.59"
+log = "0.4.21"
+env_logger = "0.11.3"
+errno = "0.3.9"
```

Comment:
The standard for this project:
* use workspace dependencies, e.g. ("libc.workspace = true")
* elide crate patch version to allow for minor updates, e.g. "errno = "0.3"